api/tests/lib/db_testing.py

```diff
@@ -18,21 +19,27 @@ def create_isolated_db(monkeypatch) -> db.DBClient:
     that connects to that schema. Drops the schema after the context manager
     exits.
     """
-    schema_name = f"test_schema_{uuid.uuid4().int}"
-    monkeypatch.setenv("DB_SCHEMA", schema_name)
+
+    schema_prefix = f"test_{uuid.uuid4().int}_"
+
+    test_schemas = [f"{schema_prefix}{schema}" for schema in Schemas]
+
+    monkeypatch.setenv("SCHEMA_PREFIX_OVERRIDE", schema_prefix)
     monkeypatch.setenv("DB_CHECK_CONNECTION_ON_INIT", "False")
 
     # To improve test performance, don't check the database connection
     # when initializing the DB client.
     db_client = db.PostgresDBClient()
     with db_client.get_connection() as conn:
-        _create_schema(conn, schema_name)
+        for schema in test_schemas:
```

Comment:
If the above change was made, we could then use the `db_config` object here instead of replicating the logic in line 25:

```suggestion
        for schema in db_config.get_schema_translate_map().values():
```